relion/convert/convert.py

```diff
@@ -387,6 +387,11 @@ def particleToRow(part, partRow, **kwargs):
     if kwargs.get('fillRandomSubset') and part.hasAttribute('_rlnRandomSubset'):
         partRow.setValue(md.RLN_PARTICLE_RANDOM_SUBSET,
                          int(part._rlnRandomSubset.get()))
+        if part.hasAttribute('_rlnBeamTiltX'):
```

Comment:
I think the row.setValue also accepts string literals, so I would prefer this way, to avoid dependencies with the xmipp Metadata class and labels required there for compilation.